tests/calculators/test_orca.py

```diff
@@ -131,15 +132,15 @@ def test_orca_geometry_optimisation(tmp_path):
     inputs = ConfigSet(atoms)
     outputs = OutputSpec()
 
-    calc = ORCA(directory=home_dir, 
+    calc = ORCA(workdir=home_dir, 
                 keep_files = "default", 
                 mult=1, 
                 task="opt")
 
 
-    generic.run(inputs=inputs, outputs=outputs, calculator=calc, properties=["energy", "forces"], output_prefix="orca_", num_python_subprocesses=0)
+    generic.run(inputs=inputs, outputs=outputs, calculator=calc, properties=["energy", "forces"], output_prefix="orca_", autopara_info=AutoparaInfo(num_python_subprocesses=0))
 
-    out = [at for at in ConfigSet(outputs)][0]
+    out = [at for at in outputs.to_ConfigSet()][0]
```

Comment:
Any reason why this isn't doing `out = generic.run(...` above? I guess we'll still need an `out = list(out)[0]` statement, but I think it'll still be more readable.

Reply:
updated!